force-app/main/default/classes/StorageChargeCheck.cls

```diff
@@ -0,0 +1,62 @@
+public with sharing class StorageChargeCheck implements Database.Batchable<sObject>
```

Comment:
You should implement Schedulable interface also in order to schedule batch daily